pylint/checkers/base/name_checker/checker.py

```diff
@@ -369,7 +340,6 @@ def visit_functiondef(self, node: nodes.FunctionDef) -> None:
                 if utils.has_known_bases(node.parent.frame(future=True))
                 else interfaces.INFERENCE_FAILURE
             )
```

Comment:
```suggestion
            )

```

New lines help with readability.